onnx/backend/test/cpp/driver_test.cc

```diff
@@ -174,6 +174,10 @@ class ONNXCppDriverTest
     }
     return true;
   }
+
+  std::vector<std::vector<uint64_t>> shape_pool;
```

Comment:
could you add some comments about these variables?

Reply:
Separate member variables from member functions?

Reply:
@yinghai  Yes I think this pool is better to be designed as a member variable of class instead of function, is it?

Reply:
@yinghai oops I know what you mean now...... okay I will separate them hhh